### PR TITLE
Add 3 new instances to ruleset

### PR DESCRIPTION
**Pull request by rocodes**
[edit]
- Ruleset has been updated to include instances belonging to investgace.cz, espenandersen.no, and nytimes.com

### Status

Ready for Review

### Review Checklist

- [x] Changes to `onboarded.txt` are accurate
- [ ] The file `default.rulesets.TIMESTAMP.gz` has been updated, extracting that file and inspecting the contents of the JSON file produces the expected rules
- [ ] The ruleset has been verified by modifying the HTTPS Everywhere configuration in a Tor Browser instance pointing to `Path Prefix`: `https://raw.githubusercontent.com/freedomofpress/securedrop-https-everywhere-ruleset/$BRANCH_NAME`
- [ ] `index.html` has been updated using `./update_index.sh`

### CODE REVIEW

**Comment by rocodes** [comment]
@emkll these ruleset changes now include rulesets for 2 instances, investigace.cz and espenandersen.no. The only changes here are the addition of the rulesets (updated `onboarded.txt`, running `sddir.py`).

When I ran `sddir.py`, it also generated a ruleset for abc.au. Not sure if that was expected, but I did not include it in this commit.

File: rulesets/new-york-times-securedrop-ruleset.xml
Comment (rocodes) [inline rulesets/new-york-times-securedrop-ruleset.xml:4]: Actually, to `https://ej3kv4ebuugcmuwxctx5ic7zxh73rnxt42soi3tdneu2c2em55thufqd.onion`, which this redirects to.

**Comment by rocodes** [comment]
A further update for @conorsch and anyone following along at home: NYT uses https on their source interface. If I should manually edit the xml to specify https instead of http, let me know and I will do so (else, Tor Browser handles the redirect).

**Comment by conorsch** [comment]
> When I ran sddir.py, it also generated a ruleset for abc.au. Not sure if that was expected, but I did not include it in this commit.

Can reproduce over here. Similarly not committing here because out of scope, but clearly related to changes in https://github.com/freedomofpress/securedrop-https-everywhere-ruleset/pull/11. Will file a separate issue to follow up.

>  If I should manually edit the xml to specify https instead of http, let me know and I will do so (else, Tor Browser handles the redirect).

No, not necessary, although we have https://github.com/freedomofpress/securedrop-https-everywhere-ruleset/issues/4 tracking the need for such special casing.

@rocodes I've appended the necessary signature for the new Onion Names, and confirmed on my machine that manually adding a new update channel tracking this PR branch successfully allows all three new Onion Names to resolve: NYT, Investigace, and Espen's.

**Comment by conorsch** [comment]
Holding off on merge so another team member can verify my results described above. The PR template says

> The ruleset has been verified by modifying the HTTPS Everywhere configuration in a Tor Browser instance pointing to Path Prefix: https://raw.githubusercontent.com/freedomofpress/securedrop-https-everywhere-ruleset/$BRANCH_NAME

and in this case, that's technically true, although only because I manually pushed a branch called "addition" to the origin remote. Since this PR is from a fork, the normal URL would be `https://raw.githubusercontent.com/rocodes/securedrop-https-everywhere-ruleset/addition`. The steps plug in a test channel for evaluation are:

1. Start Tor Browser
2. Add-ons
3. HTTPS Everywhere -> `...`
4. Preferences -> Update Channels
5. Copy/paste the settings from "SecureDropTorOnion", modifying the URL to be `https://raw.githubusercontent.com/rocodes/securedrop-https-everywhere-ruleset/addition`
6. Manually test the new Onion Names (e.g. `nytimes.securedrop.tor.onion`) and confirm redirection to the specified URLs in this PR.

**Review by conorsch — DISMISSED** [review]
Still requesting a +1 from @emkll, but looks good to me.

**Review by emkll — CHANGES_REQUESTED** [review]
It seems like there's a extra signature present in these changes that might be an artifact from a failed signature: `rulesets-signature.1607474996.sha256 `

The signature of the latest ruleset is in `rulesets-signature.1607477429.sha256 `

The changes here are good to merge once that file is deleted.

**Comment by conorsch** [comment]
> It seems like there's a extra signature present in these changes that might be an artifact from a failed signature

Great catch! Yes, that's exactly right: I cleaned up the gzip, but not the corresponding signature. Documented that oversight as a testable condition in https://github.com/freedomofpress/securedrop-https-everywhere-ruleset/issues/21#issuecomment-742142009

**Review by emkll — APPROVED** [review]
:rocket:

**Pull request merged**
